accounts/migrations/0008_auto_20170331_1702.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
Yes it does, but sometimes this might imply destroying data etc or might take a lot of time for changes in models with large number of rows. I'm not saying it is not possible, just that it's not very practical.